openmetadata-service/src/main/java/org/openmetadata/service/resources/alerts/AlertActionResource.java

```diff
@@ -410,6 +410,25 @@ public Response deleteAlertAction(
     return response;
   }
 
+  @DELETE
+  @Path("/name/{name}")
+  @Operation(
+      operationId = "deleteAlert",
+      summary = "Delete an Alert Action",
```

Comment:
Please change all the operationId to a unique name by adding `byName`. Example `deleteAlertByName`. Otherwise the operationId will conflict with the operationId of delete by Id methods.